glib/src/value.rs

```diff
@@ -454,6 +462,62 @@ impl Uninitialized for Value {
     }
 }
 
+impl ToValue for Value {
+    fn to_value(&self) -> Value {
+        self.clone()
+    }
+
+    fn value_type(&self) -> Type {
+        self.type_()
+    }
+}
+
+impl <'a> ToValue for &'a Value {
+    fn to_value(&self) -> Value {
+        (*self).clone()
+    }
+
+    fn value_type(&self) -> Type {
+        self.type_()
+    }
+}
+
+pub struct NopChecker;
+
+unsafe impl ValueTypeChecker for NopChecker {
+    type Error = std::convert::Infallible;
+    fn check(_value: &Value) -> Result<(), Self::Error> {Ok(())}
+}
+
+unsafe impl <'a> FromValue<'a> for Value {
```

Comment:
```suggestion
unsafe impl <'a> FromValue<'a> for &'a Value {
```

Too maybe

Reply:
What for? The method already takes `&self`, there is no owned version. What you suggest would take an `&'a &'a self`, which is weird.

Reply:
No, it would return an `&'a Value` instead of a `Value`

Reply:
Oh, I see.